pkg/build/builder/sti.go

```diff
@@ -354,8 +362,9 @@ func buildEnvVars(build *api.Build) s2iapi.EnvironmentList {
 // the strategy's environment. There is no preference given to either lowercase
 // or uppercase form of the variable.
 func scriptProxyConfig(build *api.Build) (*s2iapi.ProxyConfig, error) {
-	httpProxy := ""
-	httpsProxy := ""
+	var httpProxy string
+	var httpsProxy string
```

Comment:
i don't have a strong opinion either way (though i am somewhat sympathetic to @PI-Victor's position), which perhaps motivates the next thing i'm going to say, but: creating code churn due to personal preference is not ideal.  You've changed this, and now when someone else is in this file, they decide they prefer httpProxy:="" and back it goes, etc...   

I don't see a compelling reason for us to have a "standard" on which way this particular type of declaration should be done so it seems we should default to leaving it as it was written by the original author.